src/regions/ca/info/faq/26_tested-work.fr.md

```diff
@@ -1,5 +1,8 @@
 ## Mon employeur souhaite que je passe un test avant de retourner au travail car j'ai voyagé ces derniers jours. Que dois-je faire?
 
-Vous pouvez appeler la ligne de votre autorité de santé publique pour le COVID-19. Ils évalueront le risque et décideront si vous devez vous faire dépister. En attendant, assurez-vous de vous isoler.
+Vous devez contacter la ligne de votre Autorité de santé publique pour le COVID-19. Ils évalueront le risque et décideront si vous devez vous faire tester. En attendant, assurez-vous de vous isoler pendant 14 jours, à partir du jour de votre retour au Canada. Il en va de même pour les voyages vers des régions soumises à des interdictions de voyager régionales et interprovinciales qui sont en vigueur dans de nombreuses juridictions au Canada.
 
-Plus d'informations [ici](https://www.cnesst.gouv.qc.ca/salle-de-presse/Pages/coronavirus.aspx).
+Plus d'informations ici:
+
+- [La ressource fiable canadienne pour les mises à jour pour le COVID-19](https://fr.c19.ca/)
+- [Chloé pour la COVID-19](https://covid19.dialogue.co/#/)
```

Comment:
It’s odd to reference our own website, this is done a few times in this PR

Reply:
C'est ce que j'avais dit à Marie-Pierre, mais c'est que la FAQ est envoyé directement aux patients et ils ne voient pas Chloe quand ils sont dans la FAQ.

Reply:
Hmmm all good then :)